admin/spec/service/group.service.spec.js

```diff
@@ -71,6 +96,18 @@ describe('group.service', () => {
       expect(group).toEqual(groupMock)
       done()
     })
+
+    it('should return an error if schoolId or groupId are missing', async (done) => {
+      const groupId = '123456abcde'
+      const schoolId = null
+      try {
+        await groupService.getGroupById(groupId, schoolId)
+        done()
```

Comment:
Calling done from both code paths is unusual.  Either call it once at the end of the script, or don't have it in if it's not needed (as here)

Reply:
Unit tests have been updated in the parent PR. The current coverage is 97+%